cmd/operator/main.go

```diff
@@ -171,7 +171,11 @@ func init() {
 	// specified.
 	flagset.StringVar(&cfg.ReloaderConfig.Image, "prometheus-config-reloader", operator.DefaultPrometheusConfigReloaderImage, "Prometheus config reloader image")
 	flagset.StringVar(&cfg.ReloaderConfig.CPU, "config-reloader-cpu", "100m", "Config Reloader CPU request & limit. Value \"0\" disables it and causes no request/limit to be configured.")
+	flagset.StringVar(&cfg.ReloaderConfig.CPURequest, "config-reloader-cpu-req", "", "Config Reloader CPU request. Value \"0\" disables it and causes no request to be configured.")
+	flagset.StringVar(&cfg.ReloaderConfig.CPULimit, "confiig-reloader-cpu-limit", "", "Config Reloader CPU limit. Value \"0\" disables it and causes no limit to be configured.")
 	flagset.StringVar(&cfg.ReloaderConfig.Memory, "config-reloader-memory", "50Mi", "Config Reloader Memory request & limit. Value \"0\" disables it and causes no request/limit to be configured.")
+	flagset.StringVar(&cfg.ReloaderConfig.MemoryRequest, "config-reloader-memory-req", "", "Config Reloader Memory request. Value \"0\" disables it and causes no request to be configured.")
```

Comment:
Lets say you use both, we should probably at least write that config-reloader-memory supersedes this flag, correct?

Reply:
I forgot to leave a comment that I was unsure what the desired comment would be. Perhaps "Setting this flag invalidates flag `<relevant_flag>`"?